scaffolding-chef-infra/lib/linux/attributes.json

```diff
@@ -0,0 +1,5 @@
+{{#if cfg.attributes ~}}
```

Comment:
Am I reading this correctly in that the attributes are updatable via the habitat ring?

Reply:
Yes, they are.